ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_linalg.py

```diff
@@ -839,3 +839,40 @@ def test_paddle_dist(
         y=x[1],
         p=p,
     )
+
+
+# svd
+@handle_frontend_method(
```

Comment:
It should be like `handle_frontend_test` and also you've to change the parameters accordingly. Also please make sure, All tests are being passed. Thanks!